webpack.config.js

```diff
@@ -1,16 +1,37 @@
 const path = require('path')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
```

Comment:
Once the Webpack config is settled fore this let's pull most of this back to the Webpack starter repo.